### PR TITLE
Rename group node of optimizer

### PR DESCRIPTION
**Pull request by yixinglu**
as title.

### CODE REVIEW

**Review by Shylock-Hg — APPROVED** [review]

**Comment by yixinglu** [comment]
This PR could wait for other opt rules being merged.

**Review by jievince — APPROVED** [review]

File: src/optimizer/OptRule.h [hunk 1]
```diff
@@ -45,8 +45,8 @@ class OptRule {
 
 class RuleSet final {
 public:
-    static RuleSet &defaultRules();
-    static RuleSet &queryRules();
+    static RuleSet &DefaultRules();
+    static RuleSet &QueryRules();
```
Comment (CPWstatic) [inline src/optimizer/OptRule.h:49]: Emm, it seems not our naming style.

Comment (yixinglu) [reply src/optimizer/OptRule.h]: this is the singleton function same as `Status::Error()`

**Review by CPWstatic — APPROVED** [review]
LGTM.

**Review by jievince — APPROVED** [review]

**Pull request merged**